levels/fullstack.tex

```diff
@@ -44,22 +34,39 @@ \section{Legend}
 
 \section{Skills}
 
-\begin{center}
+% Alternating row colors.
 \rowcolors{2}{gray!80!gray!50}{gray!50!gray!20}
-\begin{tabular}{ | m{15em} | m{15em} | m{15em} |  m{15em} | m{15em} |  m{15em} | m{15em} | m{15em} |}
+
+% Padding for each cell.
+{\renewcommand{\arraystretch}{2}
+
+% Skills table.
+\begin{tabular}{ | C | C | C | C | C | C | C | C |}
   \hline
   \rowcolor{blue!60!black!10}
-  Level/Skill & Product Output & Data structures/Performance & Security & Communication/Writing & Networking & Fullstack Engineering & Tooling \\
+    Level/Skill &
+
+    \cellcolor{blue!25}
+    \textbf{Product Output} &
+    \textbf{Data structures/Performance} &
+    \textbf{Security} &
+    \textbf{Communication/Writing} &
+    \textbf{Networking} &
+    \textbf{Fullstack Engineering} &
+    \textbf{Tooling} \\
   \hline
     1&
 
-    Collaborates with product with designs, mockups, and demos and then
-    implements them.&
+    \cellcolor{blue!25}
+    Collaborates with product through designs, mockups, and demos of product
+    features before implementation.&
```

Comment:
I don't understand this ordering - how can you demo a product feature if you haven't implemented it yet?

Reply:
I have not been in our UX feedback meetings, but don't we quickly hack up a POC with fake data, demo it, then go do the full implementation later? If I am misunderstanding we can reword this.